src/dom-updates.js

```diff
@@ -1,44 +1,57 @@
 import {
   getBookings,
-  getTotalCost
+  getTotalCost,
+  searchByDate
 } from './bookings'
 
 import {
   yourBookings,
-  yourBookingsCost
+  yourBookingsCost,
+  dateField,
+  dashboard,
+  searchResults,
+  availableRooms
 } from './scripts'
 
 const setDashboard = (customer, allBookings, allRooms) => {
```

Comment:
There is a lot happening inside this `setDashboard` function. Here are a could things that you could try to break this up:
- Invoke the flatpicker function elsewhere (maybe right before the invocation of this function)
- separate the creation of the info you need for the HTML into a separate funciton and return all those variables in an object for later use 
- separate the creation of the HTML into another function that accepts the above info as an argument
- invoke both those functions within the `userBookings.forEach`

Reply:
- Moved flatpickr invocation to callback of window load event listener. I suppose it could go just about anywhere, but it felt weird to leave the invocation floating around outside of an event listener or function.
- Created helper function that provides object with all required room details
- Created helper functions for creating the different kinds of cards (different innerHTML whether for dashboard or search results)
- invoked functions separately inside forEach loops for display functions

I am not sure if I did it in quite the way you imagined, but all functions are certainly looking much better. I also found opportunities to correct my sad path handling in the display functions. Pushing updates results for you to  view!